.travis.yml

```diff
@@ -1,3 +1,6 @@
+sudo: false
```

Comment:
You can omit this, it's been the default for a while

Reply:
Thanks. Done.